shopinvader/views/shopinvader_variant_view.xml

```diff
@@ -57,11 +57,11 @@
         <field name="model">shopinvader.variant</field>
         <field name="arch" type="xml">
              <tree string="Product" create="0">
-                <field name="backend_id" />
-                <field name="lang_id" />
-                <field name="default_code" />
                 <field name="name" />
-                <field name="tmpl_record_id" />
+                <field name="default_code" />
+                <field name="tmpl_record_id" optional="hide" />
```

Comment:
see my other comment regarding original record fields

Reply:
This is the tree view and also not the original record, just a link to the template.
On the tree view you have everything you need from the name I think, which is the original record's name

Reply:
fine